tests/integration/src/test/java/org/apache/pulsar/tests/integration/containers/DebeziumMongoDbContainer.java

```diff
@@ -0,0 +1,33 @@
+package org.apache.pulsar.tests.integration.containers;
```

Comment:
This file missing license header, you can try command `mvn license:format`

Reply:
I have added it，please take a look again.